src/v0/destinations/google_adwords_enhanced_conversions/networkHandler.js

```diff
@@ -94,26 +77,41 @@ const getConversionActionId = async ({ method, headers, params, metadata }) => {
  * @returns
  */
 const ProxyRequest = async (request) => {
-  const { body, method, endpoint, params, metadata } = request;
-  const { headers } = request;
+  const { body, params } = request;
 
-  const conversionActionId = await getConversionActionId({ method, headers, params, metadata });
-
-  set(
-    body.JSON,
-    'conversionAdjustments[0].conversionAction',
-    `customers/${params.customerId}/conversionActions/${conversionActionId}`,
-  );
-  const requestBody = { url: endpoint, data: body.JSON, headers, method };
-  const { httpResponse: response } = await handleHttpRequest('constructor', requestBody, {
-    destType: 'google_adwords_enhanced_conversions',
-    feature: 'proxy',
-    endpointPath: `/googleAds:uploadOfflineUserData`,
-    requestMethod: 'POST',
-    module: 'dataDelivery',
-    metadata,
+  const googleAds = new GoogleAdsSDK.GoogleAds({
+    accessToken: params.accessToken,
+    customerId: params.customerId,
+    loginCustomerId: params.loginCustomerId,
+    developerToken: params.developerToken,
+  });
+  const conversionActionId = await getConversionActionId({
+    params,
+    googleAds,
   });
-  return response;
+
+  set(body.JSON, 'conversionAdjustments[0].conversionAction', `${conversionActionId}`);
+
+  const response = await googleAds.addConversionAdjustMent(body.JSON);
+  if (response.type === 'success') {
+    return {
+      success: true,
+      response: {
+        status: response.statusCode,
+        headers: response.headers,
+        data: response.responseBody,
+      },
+    };
+  }
```

Comment:
We can override the `processAxiosResponse` implementation function in the gaec/netowrk.js to handle the already processed response